README.md

```diff
@@ -14,6 +14,9 @@
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat&v=2)](https://github.com/RichardLitt/standard-readme)
 [![Makefile Included](https://img.shields.io/badge/Makefile-Supported%20-brightgreen?=flat&logo=probot&v=2)](Makefile)
 
+## Documentation
```

Comment:
This page already has `Documentation` paragraph, move sentence there

Reply:
Moved. Added CODE_OF_CONDUCT